momoko/connection.py

```diff
@@ -149,6 +167,14 @@ def __init__(self,
 
         self._last_connect_time = 0
         self._no_conn_availble_error = psycopg2.DatabaseError("No database connection available")
+        self.shrink_period = shrink_period
+        self.shrink_delay = shrink_delay
```

Comment:
need to convert `shrink_delay` to seconds (because of `time.time()` is used in other places).